resources/measures/hvac_nighttime_oa_controls/measure.rb

```diff
@@ -0,0 +1,206 @@
+    # new_space.setName(space_name)
+
+    # # echo the new space's name back to the user
+    # runner.registerInfo("Space #{new_space.name} was added.")
+
+    # # report final condition of model
+    # runner.registerFinalCondition("The building finished with #{model.getSpaces.size} spaces.")
+	
+	template = 'ComStock 90.1-2013'
+    std = Standard.build(template)
+
+    # loop through air loops
+    unitary_system_count = 0
+    li_unitary_systems = []
+    non_unitary_system_count = 0
+    li_non_unitary_systems = []
+    model.getAirLoopHVACs.sort.each do |air_loop_hvac|
+      # skip systems that are residential, use evap coolers, or are DOAS
+      next if air_loop_res?(air_loop_hvac)
+      next if air_loop_evaporative_cooler?(air_loop_hvac)
+      next if air_loop_doas?(air_loop_hvac)
+      # skip data centers
+      next if ['Data Center', 'DataCenter', 'data center', 'datacenter', 'DATACENTER', 'DATA CENTER'].any? { |word| (air_loop_hvac.name.get).include?(word) }
+      # check unitary systems
+      if air_loop_hvac_unitary_system?(air_loop_hvac)
+        unitary_system_count += 1
+        li_unitary_systems << air_loop_hvac
+      else
+        li_non_unitary_systems << air_loop_hvac
+        non_unitary_system_count += 1
+      end
+    end
+
+    # report initial condition of model
+    runner.registerInitialCondition("The building has #{unitary_system_count} unitary systems and #{non_unitary_system_count} non-unitary airloop systems applicable for nighttime operation changes. #{rtu_night_mode} is the selected nighttime operation mode.")
+
+    if (non_unitary_system_count + unitary_system_count) < 1
+      runner.registerAsNotApplicable('No applicable systems were found.')
+      return true
+    end
+
+    oa_schd_1_count = 0
+    oa_schd_op_count = 0
+    op_schd_1_count = 0
+    fan_schd_1_count = 0
+    fan_sch_op_count = 0
+
+    # make changes to unitary systems
+    li_unitary_systems.sort.each do |air_loop_hvac|
+
+      # change night OA schedule to match hvac operation schedule for no night OA
+        # Schedule to control whether or not unit ventilates at night - clone hvac availability schedule
+        next unless air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.is_initialized
+        air_loop_vent_sch = air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.get
+        air_loop_vent_sch.setName("#{air_loop_hvac.name}_night_novent_schedule")
+        next unless air_loop_hvac.airLoopHVACOutdoorAirSystem.is_initialized
+        air_loop_oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+        next unless air_loop_oa_system.minimumOutdoorAirSchedule.is_initialized
+        air_loop_oa_system.setMinimumOutdoorAirSchedule(air_loop_vent_sch)
+        oa_schd_op_count += 1
+      end
+	  
+	   # change fan operation schedule to new clone of hvac operation schedule to ensure night cycling of fans (removing any constant schedules)
```

Comment:
Something seems off with the indentations here. Should this be included in the "li_unitary_systems" loop? If that's the case, should the "end" statement on line 171 be removed?

Reply:
Yes, that's right. Sorry, I think some of the changes I'd made in the meantime didn't get pushed.